.travis.yml

```
@@ -2,11 +2,12 @@ group: travis_latest
 language: python
```

```suggestion
language: python
dist: focal
```

The default is 16.04, but that doesn't support 3.10. Hence choosing 20.04 (Focal)